### PR TITLE
Update docs module document

### PR DESCRIPTION
**Pull request by liangshiw**
Resolved : https://github.com/abpframework/abp/issues/3063

### CODE REVIEW

**Comment by maliming** [comment]
@pranavpatil19  Can you confirm it?

File: docs/zh-Hans/Modules/Docs.md [hunk 1]
```diff
@@ -26,15 +26,19 @@ ABP框架的[文档](docs.abp.io)也是使用的此模块.
...
+你也可以在浏览器中导航到 https://abp.io/get-started. 输入项目名称为 `Acme.MyProject`, 其它保持默认选项.
+
+请注意,本文档包含了 `Entity Framework Core` 提供者 不过你也可以选择 `MongoDB` 做为数据库提供者.
 
 ![创建新项目](../images/docs-module_download-new-abp-project.png)
 
 ### 2- 运行这个空项目
 
-下载项目后, 解压压缩文档并且打开 `Acme.MyProject.sln`. 你可以看到这个解决方案包含了 `Application`, `Domain`, `EntityFrameworkCore` 和 `Web` 项目. 右键选择 `Acme.MyProject.Web` 项目**设置为启动项目**.
+下载项目后, 解压压缩文档并且打开 `Acme.MyProject.sln`. 你可以看到这个解决方案包含了 `Application`, `Application.Contrawcts`, `DbMigrator`, `Domain`, `Domain.Shared`, `EntityFrameworkCore`, `EntityFrameworkCore.DbMigations`, `HttpApi`, `HttpApi.Client` 和 `Web` 项目. 右键选择 `Acme.MyProject.Web` 项目**设置为启动项目**.
```
Comment (maliming) [inline docs/zh-Hans/Modules/Docs.md]: Typo: `Contrawcts`

File: docs/zh-Hans/Modules/Docs.md [hunk 2]
```diff
@@ -176,37 +162,46 @@ ABP框架的[文档](docs.abp.io)也是使用的此模块.
...
+          /* Configure your own tables/entities inside here */
+
+          //builder.Entity<YourEntity>(b =>
+          //{
+          //    b.ToTable(QaDocConsts.DbTablePrefix + "YourEntities", QaDocConsts.DbSchema);
+
+          //    //...
+          //});
+
+          builder.ConfigureDocs();
```
Comment (maliming) [inline docs/zh-Hans/Modules/Docs.md]: I think the `ConfigureDocs `method should be located in the `XXXMigrationsDbContext `class. Right?

File: docs/en/Modules/Docs.md [hunk 3]
```diff
@@ -20,23 +20,25 @@ When you use GitHub to store your docs, Docs Module supports versioning. If you
...
 
-Note that this document covers `Entity Framework Core` provider but you can also select `MongoDB` as your database provider. 
+Note that this document covers `Entity Framework Core` provider but you can also select `MongoDB` as your database provider.
 
 ![Create new project](../images/docs-module_download-new-abp-project.png)
 
 ### 2- Running The Empty Application
 
-After you download the project, extract the ZIP file and open `Acme.MyProject.sln`. You will see that the solution consists of `Application`, `Domain `, `EntityFrameworkCore` and `Web` projects. Right click on `Acme.MyProject.Web` project and **Set as StartUp Project**.
+After you download the project, extract the ZIP file and open `Acme.MyProject.sln`. You will see that the solution consists of `Application`, `Application.Contrawcts`, `DbMigrator`, `Domain`, `Domain.Shared`, `EntityFrameworkCore`, `EntityFrameworkCore.DbMigations`, `HttpApi`, `HttpApi.Client` and `Web` projects. Right click on `Acme.MyProject.Web` project and **Set as StartUp Project**.
```
Comment (maliming) [inline docs/en/Modules/Docs.md]: Typo: `Contrawcts`

File: docs/en/Modules/Docs.md [hunk 4]
```diff
@@ -176,51 +159,57 @@ An ABP module must declare `[DependsOn]` attribute if it has a dependency upon a
...
+
+          //builder.Entity<YourEntity>(b =>
+          //{
+          //    b.ToTable(QaDocConsts.DbTablePrefix + "YourEntities", QaDocConsts.DbSchema);
+
+          //    //...
+          //});
+
+          builder.ConfigureDocs();
       }
```
Comment (maliming) [inline docs/en/Modules/Docs.md]: I think the `ConfigureDocs `method should be located in the `XXXMigrationsDbContext `class. Right?

**Comment by pranavpatil19** [comment]
> @pranavpatil19 Can you confirm it?

Yes ,let me check its working or not . Thank you for quick replay

**Comment by maliming** [comment]
@pranavpatil19 Any news?

**Comment by pranavpatil19** [comment]
> @pranavpatil19 Any news?

I will Replay as soon as possible

**Comment by liangshiw** [comment]
@pranavpatil19 The file system chapter does not need to be updated, it is still valid

**Comment by pranavpatil19** [comment]
Insert query get error

**Comment by liangshiw** [comment]
Any error? Please create issues for discussion.

**Comment by pranavpatil19** [comment]
'1)Thax For new Document Update .Work perfectly .

a) New button style is not match compare original document of abp
b)How to versioning Documents ..

[image: Screenshot_1.png]

2)When I try to integrate Git and Files system Document Model Together Its
Not working .I get 404 error
[image: Screenshot_2.png]

3) in which project  i have to implement  Full-Text Search ..please guide

Full-Text Search(Elastic Search)

The Docs module supports full-text search using Elastic Search. It is not
enabled by default. You can configure DocsElasticSearchOptions to enable it.

Configure<DocsElasticSearchOptions>(options =>
{
    options.Enable = true;
    options.IndexName = "your_index_name"; //default IndexName is abp_documents
});

The Index is automatically created after the application starts if the
Index does
not exist.

DefaultElasticClientProvider is responsible for creating IElasticClient. By
default, it reads Elastic Search's Url from IConfiguration. If your
IElasticClient needs additional configuration, please use override
IElasticClientProvider service and replace it in the dependency injection
<https://docs.abp.io/en/abp/latest/Modules/Dependency-Injection> system.

{
  "ElasticSearch": {
    "Url": "http://localhost:9200"
  }
}

**Pull request merged**